### PR TITLE
Fixed user agents downloader

### PR DESCRIPTION
**Pull request by tandriamil**
PR to fix the issue #1087.

What has been modified:
- Removed useragentstring.com as user agents source as `http://www.useragentstring.com/pages/All/` is not anymore available.
- Fixed BeautifulSoup usages (not `soup.body.[prop]` anymore but directly `soup.[prop]`).
- Fetched new UAs using the script's fixed version.

### CODE REVIEW

File: pwnlib/data/useragents/LICENSE.txt [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 The user agent strings are downloaded from:
- - http://www.useragentstring.com/
- - http://www.user-agents.org/
+ - http://www.user-agents.org
  - http://techpatterns.com/downloads/firefox/useragentswitcher.xml
 
-The latter list is part of the Firefox `User Agent Switcher' add-on and released
-under BSD 2 Clause.
+The latter list is part of the Firefox `User Agent Switcher' add-on and
+released under BSD 2 Clause.
```
Comment (zachriggle) [inline pwnlib/data/useragents/LICENSE.txt:9]: Why were these lines changed?  It doesn't matter, I'm just curious.

Comment (tandriamil) [reply pwnlib/data/useragents/LICENSE.txt:9]: [PEP8](https://www.python.org/dev/peps/pep-0008) reflex, I have this habit to limit every line to 79 chars for readability on a splitted pane  :sweat_smile:
